microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
+
+### Icon Descriptions
+
+Below you’ll find a description of each app icon and where they show up in Windows.
+
+![App Icon](../media/windows-image-app-icon.png) 
+The normal app icon that's shown in the start menu, task bar, or task manager.
+
+* 44x44
+* 55x55 (1.25x scale)
+* 66x66 (1.5x scale)
+* 88x88 (2x scale)
+* 176x176 (4x scale)
+
+![Small tile](../media/windows-image-small-tile.png) 
+The small tile is shown in the start menu when the user set your app’s tile to small size.
+
+* 71x71
+* 89x89 (1.25x scale)
+* 107x107 (1.5x scale)
+* 142x142 (2x scale)
+* 284x284 (4x scale)
+
+![Medium tile](../media/windows-image-medium-tile.png)
+The medium tile icon is shown in the start menu when the user sets your app’s tile to medium size.
+
+* 150x150
+* 188x188 (1.25x scale)
+* 225x225 (1.5x scale)
+* 300x300 (2x scale)
+* 600x600 (4x scale)
+
+![Wide tile](../media/windows-image-wide-tile.png)
+The wide tile icon
+Shown in the start menu when the user sets your app’s tile to wide size.
+
+* 310x150
+* 388x188 (1.25x scale)
+* 465x225 (1.5x scale)
+* 620x300 (2x scale)
+* 1240x600 (4x scale)
+
+![Large tile](../media/windows-image-large-tile.png)
+The large tile icon is in the start menu when the user sets your app’s tile to large size.
+
+* 310x310
+* 388x388 (1.25x scale)  
+* 465x465 (1.5x scale)
+* 620x620 (2x scale)
+* 1240x1240 (4x scale)
+
+![Store Logo](../media/windows-image-store-logo.png)
+The store logo icon is shown in app installer, Windows Partner Center, the “Report an app” option in the Store, and the “Write a review” option in the Store.
+
+* 50x50
+* 63x63 (1.25x scale)
+* 75x75 (1.5x scale)
+* 100x100 (2x scale)
+* 200x200 (4x scale)
+
+![Splash Screen](../media/windows-image-splash.png)
+The splash screen asset is shown as the splash screen for your app. Currently supported only in classic package. In the future, we may add support for the modern hosted app package as well.
+
+* 620x300
+* 775x375 (1.25x scale)
+* 930x450 (1.5x scale)
+* 1240x600 (2x scale)
+* 2480x1200 (4x scale)
+
+### Target Sized Images
+
+In addition to the standard scale factor sizes described above, we also recommend creating “target-size” assets. We call these assets target-size because they target specific sizes, such as 16 pixels, rather than specific scale factors, such as 400. Target-size assets are for Windows surfaces that don’t use the scaling plateau system.
+
+![Shown in start jump list, shortcuts, control panel:](../media/windows-image-target-size.png)
+
+* 16x16 (recommended)
+* 20x20
+* 24x24 (recommended)
+* 30x30
+* 32x32 (recommended)
+* 36x36
+* 40x40
+* 48x48 (recommended)
+* 60x60
+* 64x64
+* 72x72
+* 80x80
+* 96x96
+* 256x256 (recommended)
 
-<!-- ====================================================================== -->
```

Comment:
Please leave this html comment separator here. We have a convention on this repo where we add this separate before each H2 heading. This makes it easier to read the markdown code in VS Code.

Reply:
added before <!-- ====================================================================== -->
## Icon image sizes